sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/utils/SchemaTestUtils.java

```diff
@@ -90,4 +91,71 @@ public void describeTo(Description description) {
       description.appendText(expected.toString());
     }
   }
+
+  public static class RowFieldMatcher extends BaseMatcher<Row> {
```

Comment:
This looks nifty, but maybe made it into this PR by accident? I don't see it used anywhere